commands/curation/curationaudit.go

```diff
@@ -1134,6 +1153,32 @@ func buildNpmDownloadUrl(url, repo, name, scope, version string) []string {
 	return []string{packageUrl}
 }
 
+func getDockerNameScopeAndVersion(id, artiUrl, repo string) (downloadUrls []string, name, scope, version string) {
+	if id == "" {
+		return
+	}
+
+	id = strings.TrimPrefix(id, "docker://")
+
+	if idx := strings.Index(id, ":sha256:"); idx > 0 {
+		name = id[:idx]
+		version = id[idx+1:]
+	} else if idx := strings.LastIndex(id, ":"); idx > 0 {
+		name = id[:idx]
+		version = id[idx+1:]
+	} else {
+		name = id
+		version = "latest"
+	}
```

Comment:
can you add a comment above each with example of the expected id in this case?
consider using `switch` instead of `else if` + `else`